_posts/2020-02-20-community-build.md

```diff
@@ -0,0 +1,138 @@
+---
+layout: blog-detail
+post-type: blog
+by: Seth Tisue
+title: "Scala 2 community build achieves completion"
+---
+
+The Scala community build is a large collection of open-source Scala
+code that we build and run every day in order to validate changes to
+the Scala compiler and standard library.
+
+If you're not familiar with the community build at all, this
+[introduction](https://www.scala-lang.org/2018/01/16/community-build-growth.html),
+from January 2018, still applies.
+
+A year ago, we [updated
+you](https://www.scala-lang.org/2019/01/18/community-build.html) on
+improvements made up through January 2019.  The post you are reading
+highlights the progress the community build has made since then.
+
+In short, we believe the Scala 2 community build is now essentially
```

Comment:
You suddenly switch to "Scala _2_ community build" (which is also in the title).  Maybe either stay consistent or maybe mention something like "(as opposed to the Scala 3 community build, described later)"?  I don't mind which.  Also, just minor nit.

Reply:
Yeah.... I struggled with this. I adjusted slightly in a commit I just pushed.